frontend/awx/interfaces/InventoryGroupCreate.ts

```diff
@@ -0,0 +1,6 @@
+export interface InventoryGroupCreate {
```

Comment:
Lets consider exporting this from the InventoryGroup interface, which already exists.